app/services/WhatYouOweService.scala

```diff
@@ -43,6 +43,13 @@ class WhatYouOweService @Inject()(val financialDetailsService: FinancialDetailsS
     }
   }
 
+  val notCodedOutPoa: ChargeItem => Boolean = chargeItem => {
```

Comment:
same as above, looks like a duplicate code which we can avoid by moving it in ChargeItem?